### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by rabayarabu**
- Create a method called current_user in the ApplicationController to make current user data available to all controllers.

1. It will return the first user from the database.

- Create forms to perform the following functions:

1. Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
2. Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).

- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```
Comment (OLIPLICHE) [inline app/controllers/comments_controller.rb:1]: - Hi, it's a good practice to define a new method on `comment`. By creating a new comment, the new method can make a @comment instance variable accessible in the view:
```ruby
 def new
    @comment = Comment.new
  end
```
I would suggest you add this code above

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @rabayarabu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Create forms for use in creating posts and comments 👍 
- Add likes to posts 👍 
- Add likes to comments 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (rabayarabu) [reply app/controllers/comments_controller.rb:1]: @OLIPLICHE Thanks for your valuable suggestion. I just updated the required change. Please review my work. Thank you.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @rabayarabu 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @rabayarabu, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**